### PR TITLE
Convert Sonarr v3 quality definitions to MD table

### PR DESCRIPTION
**Pull request by rcdailey**
Eventually the screenshot should probably go away, but I've kept it in
for now to get a feel for the table. The table is important to allow for
the quality definition data to the parsed by a script.

### CODE REVIEW

**Comment by rcdailey** [comment]
Please look very carefully at the min/max numbers in the table and verify I have accurately entered them according to the screenshot. Thank you.

File: docs/Sonarr/V3/Sonarr-Quality-Settings-File-Size.md [hunk 1]
```diff
@@ -36,4 +36,23 @@ I only do WEB-DL my self for TV shows because in my opinion WEB-DL is the sweet
 
 ## Sonarr Quality Definitions
 
+| Quality            | Minimum | Maximum |
+| ------------------ | ------- | ------- |
+| HDTV-720p          | 17.9    | 45      |
+| HDTV-1080p         | 22.8    | 137.3   |
+| WEBRip-720p        | 20      | 90      |
+| WEBDL-720p         | 20      | 90      |
+| Blueray-720p       | 35      | 90      |
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-Quality-Settings-File-Size.md:45]: "Bluray"

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
other than the typo the numbers look good; I'll defer to @TRaSH-  on adding the table.  Seems like a good idea to me.

File: docs/Sonarr/V3/Sonarr-Quality-Settings-File-Size.md (hunk 1 above)
Comment (rcdailey) [reply docs/Sonarr/V3/Sonarr-Quality-Settings-File-Size.md:45]: Dammit! Thank you :)

**Review by bakerboy448 — APPROVED** [review]

**Comment by TRaSH-** [comment]
Merged with #139 being i couldn't merge my changes with yours.

**Comment by rcdailey** [comment]
For future reference, I highly recommend merging PRs as-is or requesting more changes if you'd like. Commit metadata like commit author isn't lost in that case, and things come in more incrementally. For example, in your case I would have done this instead:

1. Left feedback on this PR requesting the screenshot removed, which I would have gladly done
2. Merge my PR as-is
3. You make a follow up PR with the anime quality settings

**Pull request closed without merging**